packages/ibm-products/src/components/CreateFullPage/CreateFullPage.js

```diff
@@ -165,72 +178,88 @@ export let CreateFullPage = React.forwardRef(
         className={cx(blockClass, className)}
         {...getDevtoolsProps(componentName)}
       >
-        <div className={`${blockClass}__influencer`}>
-          <CreateInfluencer
-            stepData={stepData}
-            currentStep={currentStep}
-            title={title}
-          />
-        </div>
-        <div className={`${blockClass}__body`}>
-          <div className={`${blockClass}__main`}>
-            <div className={`${blockClass}__content`}>
-              <Form className={`${blockClass}__form`} aria-label={title}>
-                <StepsContext.Provider
-                  value={{
-                    currentStep,
-                    setIsDisabled,
-                    setOnNext: (fn) => setOnNext(() => fn),
-                    setOnMount: (fn) => setOnMount(() => fn),
-                    setStepData,
-                    stepData,
-                  }}
-                >
-                  {React.Children.map(children, (child, index) => (
-                    <StepNumberContext.Provider value={index + 1}>
-                      {child}
-                    </StepNumberContext.Provider>
-                  ))}
-                </StepsContext.Provider>
-              </Form>
-            </div>
-            <ActionSet
-              className={`${blockClass}__buttons`}
-              actions={createFullPageActions}
-              buttonSize="2xl"
-              size="2xl"
+        {(title || breadcrumbs?.length > 0) && (
+          <div className={`${blockClass}__header`} ref={headerRef}>
+            <BasicHeader
+              title={title}
+              breadcrumbs={breadcrumbs}
+              noTrailingSlash
             />
           </div>
-        </div>
-        <ComposedModal
-          className={`${blockClass}__modal`}
-          size="sm"
-          open={modalIsOpen}
-          aria-label={modalTitle}
-          onClose={() => {
-            setModalIsOpen(false);
+        )}
+        <div
+          className={`${blockClass}__influencer-and-body-container`}
+          style={{
+            height: `calc(100vh - ${headerHeight}px)`,
           }}
         >
-          <ModalHeader title={modalTitle} />
-          <ModalBody>
-            <p>{modalDescription}</p>
-          </ModalBody>
-          <ModalFooter>
-            <Button
-              type="button"
-              kind="secondary"
-              onClick={() => {
-                setModalIsOpen(!modalIsOpen);
-              }}
-              data-modal-primary-focus
-            >
-              {modalSecondaryButtonText}
-            </Button>
-            <Button type="button" kind="danger" onClick={onClose}>
-              {modalDangerButtonText}
-            </Button>
-          </ModalFooter>
-        </ComposedModal>
+          <div className={`${blockClass}__influencer`}>
+            <CreateInfluencer
+              stepData={stepData}
+              currentStep={currentStep}
+              title={secondaryTitle}
```

Comment:
This might constitute a breaking for anyone using the full page already. 🤔
The naming might not feel as straightforward but maybe we keep this as `title` and instead rename the page header title to `pageTitle` or something else.

CC @matthewgallo @davidmenendez for thoughts.

Reply:
Update from Baraa!

`title` may not be heavily used yet since while the prop existed, it was not actually in use until his last contribution in https://github.com/carbon-design-system/ibm-products/pull/3399

Reply:
I will keep it as is if there are no objections.